docker-compose.lambda-zips.js.yml

```diff
@@ -1,14 +0,0 @@
-version: "3.8"
```

Comment:
Changes to this file are used to determine if we need to build the JS lambda in our merge pipeline. I would hate to lose this ability; I don't want to be building new versions of services if we don't need to. This takes up time in the pipeline, space in artifact repositories, and leads to unnecessary churn when deploying services to production.

As it is, we do this with our Rust services because we don't yet have a good way to determine which of them may (or may not) have changed in a given PR. I have some notes on how to do this, though; once that's in place we would only build what we needed to for any given run through the pipeline.

Reply:
In that case it seems the Make build targets should be broken up and rearranged different to include thing like a `build-rust` and `build-graphql-endpoint` (as well as others).

It seems the merge pipeline has different upload steps based on type of artifact, but it seems it'd be more appropriate for that to match the build targets, so we'd have one for Rust and one for graplql-endpoint.